src/assignments/2-operator.js

```diff
@@ -14,6 +14,31 @@ output example
 */
 // start here writing the code for program1
 
+var num1 = 30;
+var num2 = 45;
+var add = num1 + num2;
+console.log(`${num1} + ${num2} = ${add}`)
+
+var num1 = 30;
+var num2 = 45;
+var substract = num1 - num2;
+console.log(`${num1} - ${num2} = ${substract}`)
+
+var num1 = 30;
+var num2 = 45;
+var multiply = num1 * num2;
+console.log(`${num1} * ${num2} = ${multiply}`)
+
+var num1 = 30;
+var num2 = 45;
+var divide = num1 / num2;
```

Comment:
don't use var for the next time, instead of using let. 
before divide the number, you should check whether num2 > 0 or not, then do the divide calculation.